docs/guides/linux-containers.rst

```diff
@@ -9,24 +9,24 @@ Using Linux containers
 Host requirements
 -----------------
 
-**Your host must be Debian based**
+**Your host must be Debian based.**
 
-It can be Ubuntu 14.x, Debian Wheezy/Jessie, etc.
+It can be Ubuntu 14.04 or later, or Debian Wheezy or later.
```

Comment:
I would say that Wheezy is already phasing out. DebOps still works well with Jessie, but Stretch is the main target now.

Perhaps you could rephrase it to say that DebOps is targeted for the current Debian Stable release, with Oldstable being supported as well as long as it's supported by Debian. That way we can avoid bumping it every two years. :-)